app/assets/javascripts/helpers/calendar_helper.es6.jsx

```diff
@@ -41,14 +59,15 @@ function availability_to_events(availability, timezone) {
 }
 
 function number_to_moment(number, timezone) {
+
   var day = Math.floor(number/96);
   var hour = Math.floor((number%96)/4);
   var minute = ((number%96)%4)*15;
-  var m = moment().utc().startOf('week').add(day, 'days').add(hour, 'hours').add(minute, 'minutes');
-  return moment.tz(m, timezone);
+
+  return moment.tz(timezone).startOf('week').add(day, 'days').add(hour, 'hours').add(minute, 'minutes')
 }
 
-function get_unavailable_availability(availability) {
+function get_unavailable_availability(availability, timezone) {
```

Comment:
Seems like we don't need the timezone argument anymore. Can probably just get rid of it.